### DISCUSSION TITLE
Material Scrollbar

### DISCUSSION THREAD

**Discussion by iamzaidsoomro** [twbs/bootstrap#36334] [Ideas] [answered]
Please add the custom scrollbar with material colors.

**Comment by EsperLazer** [comment]
I don't know if this answer is what you were looking for, but the following method will change the color of the scrollbar.

Make changes to the "style" of the page where you want to change the color of the scrollbar.<br>

<style><br>
/**/<br>
::-webkit-scrollbar {<br>
  width: 10px;<br>
  height: 10px;<br>
}<br>
<br>
::-webkit-scrollbar-track {<br>
  background: #ffe6ea;<br>
}<br>
<br>
::-webkit-scrollbar-thumb {<br>
  background: #ff8095;<br>
}<br>
/**/<br>
</style><br>

**Comment by mdo** [comment, marked as answer]
Just to give this an explicit answer—we have no plans to do that :).
